strap.sh

```diff
@@ -50,15 +63,21 @@ fetch_keyring()
 # note: this is pointless if you do not verify the key fingerprint
 verify_keyring()
 {
-    gpg \
-        --keyserver pgp.mit.edu \
-        --recv-keys 4345771566D76038C7FEB43863EC0ADBEA87E4E3 > /dev/null 2>&1
+    if ! gpg --keyserver pgp.mit.edu \
```

Comment:
you used if; then notation in the rest of the file. Wouldn't it be good to keep it consistent

Reply:
You are correct. What I thought was: when the condition have multiple lines I think the `then` in a new line makes it easier to the eyes. What do you think?

Reply:
I prefer ```if;then``` over ```then``` in a newline, but for better readability i prefer a consistent style. 
How about using ```then``` in a new line file wide, as i see your point for multi-line conditionals?

Reply:
Perfect and I agree. I'll make the changes later. Thanks @ikstream